sites/friday/src/add-services/clickhouse.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "ClickHouse"
+weight: -110
+description: 
+---
+
+ClickHouse is a high-performance column-oriented, distributed, OLAP (Online Analytical Processing) database.</br>
+It allows you to generate real-time analytical data reports using SQL queries.</br>
+For more information, see the [ClickHouse documentation](https://ClickHouse.com/docs).
+
+{{% note %}} 
+
+{{% vendor/name %}} supports ClickHouse with the following limitations:
+
+- High availibility of service isn't supported.
+- You can only configure single-node ClickHouse clusters.
+
+{{% /note %}} 
+
+## Supported versions
+
+- 23.8
```

Comment:
is it as all of the other version? do we write it in stone or do we use the register.json file to display it ?